packages/@uppy/aws-s3/src/index.js

```diff
@@ -249,25 +247,18 @@ export default class AwsS3 extends UploaderPlugin {
     return Promise.resolve()
   }
 
-  connectToServerSocket (file) {
-    return this.#uploader.connectToServerSocket(file)
-  }
-
-  #requestSocketToken = async (file) => {
+  #getCompanionClientArgs = (file) => {
     const opts = this.#uploader.getOptions(file)
-    const Client = file.remote.providerOptions.provider ? Provider : RequestClient
-    const client = new Client(this.uppy, file.remote.providerOptions)
     const allowedMetaFields = Array.isArray(opts.allowedMetaFields)
       ? opts.allowedMetaFields
       // Send along all fields by default.
       : Object.keys(file.meta)
-
+    // TODO: do we need tus in aws-s3?
```

Comment:
We most likely don't, looks like a copy-paste refactor mistake https://github.com/transloadit/uppy/commit/2c75ddacfe33d847cfe18db01509cff77385ea7f#diff-fa367bfcefda1d5a80deb4ca70a03be09d792a47e1e4070db71347ae54f56f45L268